docs/install_guides/configurations-for-intel-gpu.md

```diff
@@ -46,24 +30,31 @@ You've completed all required configuration steps to perform inference on proces
 
 ## Windows
 
-This section will help you check if you require driver installation. Install indicated version or higher.
-
-If your applications offload computation to **Intel® Integrated Graphics**, you must have the Intel Graphics Driver for Windows installed on your hardware.
-[Download and install the recommended version](https://downloadcenter.intel.com/download/30079/Intel-Graphics-Windows-10-DCH-Drivers). 
+To install the Intel Graphics Driver for Windows on your hardware, please proceed with the [instruction](https://www.intel.com/content/www/us/en/support/articles/000005629/graphics.html). 
 
 To check if you have this driver installed:
 
 1. Type **device manager** in your **Search Windows** box and press Enter. The **Device Manager** opens.
-
-2. Click the drop-down arrow to view the **Display adapters**. You can see the adapter that is installed in your computer:
-   ![](../img/DeviceManager.PNG)
-
+2. Click the drop-down arrow to view the **Display adapters**. You can see the adapter that is installed in your computer:  
+![](../img/DeviceManager.PNG)
 3. Right-click the adapter name and select **Properties**.
+4. Click the **Driver** tab to see the driver version.  
+![](../img/DeviceDriverVersion.PNG)
 
-4. Click the **Driver** tab to see the driver version. 
-   ![](../img/DeviceDriverVersion.PNG)
 
 You are done updating your device driver and are ready to use your GPU.
+
+## Additional info
+
+In the internal OpenVINO validation the following versions of Intel Graphics Driver were used:
+
+Operation System | Driver version
+--- |-------------------------
+Ubuntu 20.04 | [22.35.24055](https://github.com/intel/compute-runtime/releases/tag/22.35.24055)
+Ubuntu 18.04 | [21.38.21026](https://github.com/intel/compute-runtime/releases/tag/21.38.21026)
+CentOS 7 | [19.41.14441](https://github.com/intel/compute-runtime/releases/tag/19.41.14441)
+RHEL 8 | [22.28.23726](https://github.com/intel/compute-runtime/releases/tag/22.28.23726)
```

Comment:
Since we list all the OSes here, we should add Windows as well. But this can be fixed later. Other than than LGTM